src/spyglass/utils/dj_mixin.py

```diff
@@ -258,56 +258,32 @@ def fetch_pynapple(self, *attrs, **kwargs):
 
     # ------------------------ delete_downstream_parts ------------------------
 
-    def _import_part_masters(self):
-        """Import tables that may constrain a RestrGraph. See #1002"""
-        from spyglass.common.common_ripple import (
-            RippleLFPSelection,
-        )  # noqa F401
-        from spyglass.decoding.decoding_merge import DecodingOutput  # noqa F401
-        from spyglass.decoding.v0.clusterless import (  # noqa F401
-            UnitMarksIndicatorSelection,
-        )
-        from spyglass.decoding.v0.sorted_spikes import (  # noqa F401
-            SortedSpikesIndicatorSelection,
-        )
-        from spyglass.decoding.v1.core import PositionGroup  # noqa F401
-        from spyglass.lfp.analysis.v1 import LFPBandSelection  # noqa F401
-        from spyglass.lfp.lfp_merge import LFPOutput  # noqa F401
-        from spyglass.linearization.merge import (  # noqa F401
-            LinearizedPositionOutput,
-            LinearizedPositionV1,
-        )
-        from spyglass.mua.v1.mua import MuaEventsV1  # noqa F401
-        from spyglass.position.position_merge import PositionOutput  # noqa F401
-        from spyglass.ripple.v1.ripple import RippleTimesV1  # noqa F401
-        from spyglass.spikesorting.analysis.v1.group import (  # noqa F401
-            SortedSpikesGroup,
-        )
-        from spyglass.spikesorting.spikesorting_merge import (  # noqa F401
-            SpikeSortingOutput,
-        )
-        from spyglass.spikesorting.v0.figurl_views import (  # noqa F401
-            SpikeSortingRecordingView,
-        )
+    def _load_shared_schemas(self):
```

Comment:
could we add an additional option to pass something like `additional_prefixes = [ ]`  that gets extended onto the `all_shared` list?  This would probably be more a developer feature to know about, but would be helpful if need to load tables into the graph for a team project or debugging a different user's tables

Reply:
Sure, I can do that. The portion that calls this as part of `delete_downstream_parts` will get cut when we update the datajoint pin, but this can be left behind as a developer tool